tests/create-action.test.js

```diff
@@ -0,0 +1,112 @@
+// Remove double quotes around snapshots.
+// This improves snapshot readability for generated file contents.
+expect.addSnapshotSerializer({
+  test: value => typeof value === 'string',
+  print: value => value
+})
+
+jest.mock('fs')
+jest.mock('enquirer')
+
+let mockFs
+
+const runCLI = (...args) => {
+  const createAction = require('../bin/create-action')
+  return createAction(args)
+}
+
+beforeEach(() => {
+  jest.resetModules()
+  mockFs = require('fs')
+
+  process.exit = jest.fn()
+  console.log = jest.fn()
+  console.error = jest.fn()
+  process.cwd = jest.fn(() => '.')
+})
+
+afterEach(() => {
+  jest.restoreAllMocks()
+})
+
+test('prints help when no arguments are passed', async () => {
+  await runCLI()
+
+  expect(process.exit).toHaveBeenCalledWith(1)
+  expect(console.log).toHaveBeenCalledWith(
+    expect.stringMatching(/Usage: npx actions-toolkit <name>/)
```

Comment:
@JasonEtco What are your thoughts on updating these test assertions? What's important to assert with the stdout logging?

`createAction()` could take an optional second argument, which is a logger object (defaults to Signale), and I could add assertions for specific methods to be called in tests (sort of like I have now with `console.log`).

Maybe we could assert that the contents of `process.stdout` match a snapshot?

Not sure what the best approach is here as I update the tests.

Reply:
I generally much prefer passing a logger instance that we can mock - it lets us use snapshots really easily. So I'd say `createAction()` with a logger option!